workers/unity/Assets/Playground/Scripts/Worker/AndroidClientWorkerConnector.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using UnityEngine;
+#if UNITY_ANDROID
+using Improbable.Gdk.Mobile.Android;
+#endif
+
+namespace Playground.Worker
+{
+    public class AndroidClientWorkerConnector : AbstractMobileClientWorker
+    {
+        protected override string GetHostIp()
+        {
+#if UNITY_ANDROID
+            var hostIp = GetIpFromField();
+            if (Application.isMobilePlatform && DeviceInfo.IsAndroidStudioEmulator() && hostIp.Equals(string.Empty))
+            {
+                return DeviceInfo.AndroidStudioEmulatorDefaultCallbackIp;
+            }
+
+            return hostIp;
+#else
+            throw new NotImplementedException("Incompatible platform: Please use Android");
```

Comment:
I'm not sure `NotImplementedException` is the correct one here

Reply:
Similar comment in `IosClientWorkerConnector`

Reply:
+1

Reply:
yeah, that looks like the wrong exception

Reply:
and it also doesn't give much information. how do I use android?

Reply:
Link to Amazon page to buy an Android phone :trollface:

Reply:
that tells me how to buy an android phone, not how to use it :trollface: